lib/utils/auto-scenario-config-for-ember.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+var semver = require('semver');
+
+var EmberVersions = ['v2.5.0-beta.1', 'v2.4.0', 'v2.4.0-beta.3', 'v2.3.1', 'v2.4.0-beta.2', 'v2.4.0-beta.1', 'v2.3.0', 'v2.3.0-beta.3', 'v2.3.0-beta.2', 'v2.3.0-beta.1', 'v2.2.0', 'v2.1.1', 'v1.13.11', 'v2.2.0-beta.2', 'v2.2.0-beta.1', 'v2.1.0', 'v2.1.0-beta.4', 'v1.13.10', 'v2.0.2', 'v2.1.0-beta.3', 'v2.1.0-beta.2', 'v2.0.1', 'v1.13.9', 'v2.1.0-beta.1', 'v2.0.0', 'v1.13.8', 'v2.0.0-beta.5', 'v1.13.7', 'v2.0.0-beta.4', 'v1.13.6', 'v2.0.0-beta.3', 'v1.13.5', 'v1.13.4', 'v2.0.0-beta.2', 'v1.13.3', 'v1.13.2', 'v1.13.1', 'v1.13.0', 'v1.12.1', 'v1.13.0-beta.1', 'v1.12.0', 'v1.11.1', 'v1.12.0-beta.1', 'v1.11.0', 'v1.11.0-beta.5', 'v1.11.0-beta.4', 'v1.11.0-beta.3', 'v1.11.0-beta.2', 'v1.11.0-beta.1', 'v1.10.0', 'v1.10.0-beta.4', 'v1.10.0-beta.3', 'v1.10.0-beta.1', 'v1.9.1', 'v1.10.0-beta.2', 'v1.9.0', 'v1.9.0-beta.4', 'v1.9.0-beta.3', 'v1.9.0-beta.1', 'v1.8.1', 'v1.8.0', 'v1.7.1', 'v1.8.0-beta.5', 'v1.8.0-beta.4', 'v1.8.0-beta.3', 'v1.8.0-beta.2'];
```

Comment:
The rate limiting might be a problem at some companies/conferences. 

The package idea is interesting, especially since we could just script updating it, but it would only be updated anytime someone `npm installed`.